packages/nextjs/components/MemberCard.tsx

```diff
@@ -0,0 +1,67 @@
+import Image from "next/image";
+import Link from "next/link";
+import { Address } from "./scaffold-eth";
+
+export interface Member {
+  address: string;
+  name: string;
+  avatar: string;
+  description: string;
+  github?: string;
+  telegram?: string;
+  linkedin?: string;
+}
+
+export const MemberCard = ({ member }: { member: Member }) => (
+  <Link href={`/builders/${member.address}`} className="w-full sm:w-72">
```

Comment:
As I mentioned to you in TG:
> I think it would be better to have a button
or link
because you already have other links

What I meant is: You can't have a link around other links. There should be a button next to the other existing buttons to allow the user to navigate to the builder profile.

There are errors in the browser log related to this:
> hook.js:608 In HTML, <a> cannot be a descendant of <a>.
This will cause a hydration error.